tests/README.md

```diff
@@ -0,0 +1,94 @@
+# Yellowbrick Tests
+
+*Welcome to the Yellowbrick tests!*
+
+If you're looking for information about how to use Yellowbrick, for our contributor's guide, for examples and teaching resources, for answers to frequently asked questions, and more, please visit the latest version of our documentation at [www.scikit-yb.org](https://www.scikit-yb.org/). 
+
+## Running Yellowbrick Tests
+
+To run the tests locally, first install the tests-specific requirements with `pip` using the `requirements.txt` file in the `tests` directory:
+
+```
+$ pip install -r tests/requirements.txt
+```
+
+The required dependencies for the test suite include testing utilities and libraries such as `pandas` and `nltk` that are not included in the core dependencies.
+
+Tests can then be run as follows from the project `root`:
+
+```bash
+$ make test
+```
+
+The Makefile uses the `pytest` runner and testing suite as well as the coverage library. 
+
+## Adding a Test for Your Visualizer
+
+The `tests` package mirrors the yellowbrick package in structure and also contains several helper methods and base functionality. To add a test to your visualizer, find the corresponding file to add the test case, or create a new test file in the same place you added your code.
+
+### Visual Tests
+
+The primary test you should create is simply to test your visualizer from end to end and make sure that no exceptions occur. 
+
+Visual tests are notoriously difficult to create --- how do you test a visualization or figure? Moreover, testing scikit-learn models with real data can consume a lot of memory. To assist with this, we have two primary helpers, `VisualTestCase` and the `yellowbrick.datasets` module. 
+
+Leverage these helpers to create your unittest as follows:
+
+```python
+import pytest
+
+from tests.base import VisualTestCase
+from yellowbrick.datasets import load_occupancy
+
+
+class MyVisualizerTests(VisualTestCase):
+
+    def test_my_visualizer(self):
+        """
+        Test MyVisualizer on a real dataset
+        """
+        # Load the data using the Yellowbrick datasets module
+        X, y = load_occupancy()
+
+        try:
+            visualizer = MyVisualizer()
+            visualizer.fit(X)
+            visualizer.poof()
```

Comment:
To avoid generating warnings about the `Agg` backend, we're moving toward `visualizer.finalize()` rather than `visualizer.poof()` in the tests, this avoids an unnecessary call to `plt.show` and also protects us from some backend errors that may occur on CI with the image similarity tests. I was planning to deal with this as part of the audit and move to the new datasets in the tests. Would you mind changing this line and line 76 below to reflect this?

Reply:
Will do

Reply:
Thank you!